Client/src/components/contanier/student/StudentCourses/staticData.js

```diff
@@ -0,0 +1,19 @@
+const data = [
+  {
+    id:0,
+    titel: 'Writing & Speaking',
+    desc: 'In this course you will learn the fundemtal of speaking and writign skills',
+    date: '1/5/2019',
+    days: 'Sat - Mon - Sun',
```

Comment:
I think it should be an array ,
as can you get it from your API

Reply:
it's already an array

Reply:
I talked about the 
```
days: 'Sat - Mon - Sun',
```
is that an array :smile: !?